lxd/patches.go

```diff
@@ -3714,3 +3713,65 @@ func patchUpdateFromV30(_ *sql.Tx) error {
 
 	return nil
 }
+
+//  renames any config incorrectly set config file entries due to the lvm.thinpool_name typo
+func patchThinpoolTypoFix(name string, d *Daemon) error {
```

Comment:
@AndrewElvisDeng please can you move this function above `patchMoveBackupsInstances` so that it is the most recently added patch (they are added to the file in reverse order). Thanks